Sources/Cocoa/Components/Picker/DatePicker.swift

```diff
@@ -47,17 +48,19 @@ final class DatePicker: DrawerScreen.Content, Appliable {
     static func present(
         initialValue date: Date? = nil,
         configuration: Configuration<UIDatePicker>? = nil,
+        caller: Any? = nil,
         _ callback: @escaping (Date?) -> Void
     ) {
         let picker = DatePicker().apply {
             $0.initialDate = date
             $0.pickerView.date = date ?? Date()
+            $0.caller = caller
             $0.didChangeValue = callback
             if let configuration = configuration {
                 $0.pickerView.apply(configuration)
             }
         }
 
-        DrawerScreen.present(picker)
+        DrawerScreen.present(picker, caller: caller)
```

Comment:
Just thought here:

We are setting `caller` to `DatePicker` and then we are populating the same caller to `DrawerScreen` - as a developer who sees this for the first time I have red flag here - Do we have a way how to merge these two? So we will not populate the same item twice? (maybe not, I haven't been digging into this)

Reply:
Good catch, it was unnecessary to assign it twice.